frontend/pages/ManageControlsPage/ManageControlsPage.tsx

```diff
@@ -99,6 +99,14 @@ const ManageControlsPage = ({
     [location, router]
   );
 
+  // If MDM not configured, override title that shows up on browser tabs to generic title
+  useEffect(() => {
+    // e.g., Manage macOS hosts | Fleet for osquery
+    if (!config?.mdm.enabled_and_configured) {
+      document.title = "Manage macOS hosts | Fleet for osquery";
+    }
+  }, [location.pathname, config?.mdm.enabled_and_configured]);
```

Comment:
Ahh! I had to change the dependency to location and config, it wasn't picking up the changes that MDM is not configured otherwise, thank you!

Reply:
Hrm, it's still a little funky: 
https://github.com/fleetdm/fleet/assets/61553566/0f6c1bc2-75ac-4915-b2ea-39f5fab054b0

The issue of the flashing from one title to another might be solved by taking this logic out of the `useEffect` altogether – I'm a little rusty on exactly how `location` interacts with rendering the subnav options on this page, and don't have MDM currently configured, so not sure if that's actually kosher or if it breaks the subnavigation functionality.

Another idea is to get this logic into `App.tsx`, so there's no back-and-forth defining the title between those two layers. Seems like there's some slightly tricky async state to grapple with if going that route.

Reply:
Yeah, I don't like the flashing, but it's because we don't know that it's not enabled at the app level, only at the page level when the config API is called.

@jacobshandling Do you think I should just move all the logic to the page level then so it doesn't flash?

Reply:
Would it work to put the logic within `fetchConfig`, after awaiting the API call? So, right before [here](https://github.com/fleetdm/fleet/blob/main/frontend/components/App/App.tsx#L66)

Reply:
Perhaps abstracted to a `setDocumentTitle` function to keep it clean?